ui/src/utils/init.js

```diff
@@ -151,7 +151,7 @@ export default async (app, routes, stores, translations, additionalTranslations
         await loadLocaleMessages(i18n, locale, additionalTranslations);
         await setI18nLanguage(i18n, locale);
     }
-    
+    app.config.globalProperties.$v_i18n = i18n;
```

Comment:
if you do not need the change to `BasicSettings.vue`, you don't need this one either.